include/geode/georepresentation/io/zip_file.h

```diff
@@ -0,0 +1,64 @@
+/*
```

Comment:
Is this file in the right lib ? Is it related to basic, mesh or georepresentation ?

Reply:
It is a good question. I put it in georepresentation since it is in the only lib minizip was used for the moment. Maybe its place is in basic. What do you think of this?

Reply:
I think basic might be a better place.

Reply:
OK I move it into basic lib